### PR TITLE
Bump minor version

### PR DESCRIPTION
**Pull request by kernelwhisperer**
## What changed?
Insert here the changes.

### CODE REVIEW

**Comment by koalalorenzo** [comment]
Changes:

- Add row selection to StorageList #150
- Updates Readme, Deb and Dependencies #157
- Upgrade IPFS to v0.4.16 #158 
- Upgrade ipfs binaries to v0.4.17 #167
- Fix the files tab on the property window #159
- Track app updates #160
- Add copy to clipboard to details window #161
- Upgrade dependencies #164
- Adds debian repo and triggers #166 ( get.siderus.io )
- Make import from hash an activity #165
- Setup tray icon after the app started, not before #170 ( Closed #156 )
- Persist activities to a file #169

**Comment by koalalorenzo** [comment]
Jumping from 0.4.15 to 0.4.17 requires migration: 

```
Running /Users/koalalorenzo/Repositories/Siderus/Orion/go-ipfs/ipfs [ 'config', 'Addresses.API' ]
/Users/koalalorenzo/Repositories/Siderus/Orion/go-ipfs/ipfs config,Addresses.API: Error: ipfs repo needs migration
```

The app won't start and goes into timeout.

**Comment by 0x6431346e** [comment]
Finished the repo migration feature, to test in dev:
- first delete the current repo (`rm -rf ~/.config/Electron/ipfs-repo`)
- run `IPFS_PATH=~/.config/Electron/ipfs-repo ipfs init` with an older binary (my globally installed ipfs is 0.4.14)
- running `IPFS_PATH=~/.config/Electron/ipfs-repo ipfs repo stat` shows  `Version:    fs-repo@6`
- `make run` will now start Orion and before starting the daemon it will run migrations:

```
Running /home/daniel/repos/siderus/orion/fs-repo-migrations/fs-repo-migrations -y { env: { IPFS_PATH: '/home/daniel/.config/Electron/ipfs-repo' } }
Finished /home/daniel/repos/siderus/orion/fs-repo-migrations/fs-repo-migrations -y with:  Found fs-repo version 6 at /home/daniel/.config/Electron/ipfs-repo
===> Running migration 6 to 7...
applying 6-to-7 repo migration
migrating IPNS record for key: self
updated version file
===> Migration 6 to 7 succeeded!
```

File: Makefile [hunk 1]
```diff
@@ -73,19 +81,32 @@ _prepkg: dep _test_variables
 	NODE_ENV=${NODE_ENV} ./node_modules/.bin/electron-compile app
 .PHONY: _prepkg
 
-# Download the IPFS binary from the URL
+prepare_binaries:
```
Comment (koalalorenzo) [inline Makefile]: write like this:

```
prepare_binaries: prepare_ipfs_bin prepare_repo_migrations_bin
.PHONY: prepare_binaries
```

File: app/index.js [hunk 2]
```diff
@@ -217,6 +220,7 @@ function startOrion () {
           })
           // Then change the json of the configuration file
           .then(ensureDaemonConfigured)
+          .then(ensureRepoMigrated)
```
Comment (koalalorenzo) [inline app/index.js]: Run this before the daemon configured.

**Review by koalalorenzo — CHANGES_REQUESTED** [review]

**Review by koalalorenzo — APPROVED** [review]
It works! :D

**Pull request merged**